bugbot/rules/perfalert_resolved_regression.py

```diff
@@ -87,6 +104,63 @@ def should_needinfo(self, bug_comments, status_time):
 
         return True
 
+    def get_resolution_comments(self, comments, status_time):
+        resolution_comment = None
+        preceding_comment = None
+
+        for comment in comments:
+            if comment["creation_time"] > status_time:
+                break
+            if comment["creation_time"] == status_time:
+                resolution_comment = comment
+            if (
+                comment["author"] != BOT_MAIN_ACCOUNT
+                and comment["author"] != "intermittent-bug-filer@mozilla.bugs"
+            ):
+                preceding_comment = comment
+
+        return resolution_comment, preceding_comment
+
+    def get_resolution_comment(self, comments, bug_history):
+        status_time = bug_history["status_time"]
+        resolution_comment, preceding_comment = self.get_resolution_comments(
+            comments, status_time
+        )
+
+        if (
+            resolution_comment
+            and resolution_comment["author"] == bug_history["status_author"]
+        ):
+            # Accept if status author provided a comment at the same time
+            return resolution_comment["text"]
+        elif preceding_comment:
```

Comment:
We could use `if` instead of `elif` since it is coming after a return (the same for other cases below):
```suggestion

        if preceding_comment:
```